environments/development/postgresql.yml

```diff
@@ -11,3 +11,9 @@ postgres_override:
 dbs:
   synclogs: null
   form_processing: null
+  ucr:
+    host: citusdb1
+  ucr:
+    host: citusdb2
+  ucr:
+    host: citusdb3
```

Comment:
how come these are repeated so 3 times? Is it just to get the DB created? If that's the reason I think it would be better to have the citus role import the `set_up_dbs` task file to do the necessary setup on the workers and just leave the coordinator here. 

Also I you can't have multiple keys with the same name.

Reply:
Are you saying to use this to create DB on coordinator and write a separate task to create DB on workers?

Reply:
yes